sys-devel/gcc/gcc-5.4.0_2016_06_04.recipe

```diff
@@ -124,6 +124,7 @@ BUILD()
 		--with-bug-url=http://dev.haiku-os.org/ \
 		--with-default-libstdcxx-abi=gcc4-compatible \
 		--enable-libssp \
+		--enable-libstdcxx-filesystem-ts \
```

Comment:
Isn't that's the default? From gcc [docs](https://gcc.gnu.org/onlinedocs/libstdc++/manual/configure.html)

> `--enable-libstdcxx-filesystem-ts`[default]
>
>    Build `libstdc++fs.a` as well as the usual libstdc++ and libsupc++ libraries. This is enabled by default on select POSIX targets where it is known to work and disabled otherwise.